cmd/iam_apikey_operations.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/exoscale/egoscale"
+	"github.com/spf13/cobra"
+)
+
+type apiKeyOperationsItemOutput struct {
+	Compute []string `json:"compute,omitempty"`
+	DNS     []string `json:"dns,omitempty"`
+	IAM     []string `json:"iam,omitempty"`
+	SOS     []string `json:"sos,omitempty"`
+}
+
+func (o *apiKeyOperationsItemOutput) toJSON()  { outputJSON(o) }
+func (o *apiKeyOperationsItemOutput) toText()  { outputText(o) }
+func (o *apiKeyOperationsItemOutput) toTable() { outputTable(o) }
+
+// apiKeyShowCmd represents the API key showing command
+var apiKeyOperationsCmd = &cobra.Command{
+	Use:   "operations [Filter ...]",
```

Comment:
```suggestion
	Use:   "operations [filter ...]",
```

Please document the `filter` argument in the command long help field.